public/app/plugins/panel/timeseries/migrations.test.ts

```diff
@@ -40,13 +40,51 @@ describe('Graph Migrations', () => {
     expect(panel).toMatchSnapshot();
   });
 
-  it('legend', () => {
-    const old: any = {
-      angular: legend,
-    };
-    const panel = {} as PanelModel;
-    panel.options = graphPanelChangedHandler(panel, 'graph', old);
-    expect(panel).toMatchSnapshot();
+  describe('legend', () => {
+    test('without values', () => {
+      const old: any = {
+        angular: {
+          legend: {
+            show: true,
+            values: false,
+            min: false,
+            max: false,
+            current: false,
+            total: false,
+            avg: false,
+          },
+        },
+      };
+      const panel = {} as PanelModel;
+      panel.options = graphPanelChangedHandler(panel, 'graph', old);
+      expect(panel).toMatchSnapshot();
```

Comment:
super nit: I'm not a big fan of snapshots because:

- I need to switch context from the test to see what the expected should be
- It's way too easy to just update snapshots

I guess there were already snapshots in this file so this is OK.